pkg/assets/builder.go

```diff
@@ -22,21 +22,25 @@ import (
 	"os"
 	"strings"
 
+	"k8s.io/kops/pkg/apis/kops"
 	"k8s.io/kops/pkg/kubemanifest"
 )
 
 // AssetBuilder discovers and remaps assets
 type AssetBuilder struct {
-	Assets []*Asset
+	Assets      []*Asset
+	ClusterSpec *kops.ClusterSpec
```

Comment:
I am wondering if this should not be the full ClusterSpec, but this is internal so we can go with it...